samza-core/src/test/java/org/apache/samza/zk/TestZkBarrierForVersionUpgrade.java

```diff
@@ -74,6 +75,7 @@ public static void teardown() {
     zkServer.teardown();
   }
 
+  @Ignore("The test is flaky, see SAMZA-1193")
```

Comment:
Can you please comment out the @Test annotation instead of using Ignore annotation?
I believe last time @santhoshvenkat1988 tried with Ignore, it caused some failures in hudson. We decided that commenting out test annotation and adding a TODO with JIRA number was cleaner. You can talk to @santhoshvenkat1988 or @jmakes on why it failed on hudson.